bors.toml

```diff
@@ -0,0 +1,18 @@
+# List of commit statuses that must pass on the merge commit before it is
+# pushed to master.
+status = [
+  "continuous-integration/travis-ci/push",
```

Comment:
Should this be `continuous-integration/travis-ci/pr`? I don't think all PRs get `/push`.

Reply:
No. The way bors works is it creates a test branch it owns (using the commits from the PR, but otherwise wholly independent from the travis runs on the PR that github shows) and pushes that branch. Travis runs a job on the 'pushed' branch, which must pass.

The PR test is the test that runs when the PR is created, which is not the test we're interested in.